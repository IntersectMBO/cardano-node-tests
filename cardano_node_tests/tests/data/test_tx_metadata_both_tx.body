{
    "type": "TxBodyMary",
    "description": "",
    "cborHex": "83a400818258209d43c61d25e780112200a023840b2fdfdf318c12acc9b230685f788167a9419f00018182581d609545290549fe29e2c3e4905118d625bc0695779814551e948185146e1a002adfda021a0002e6e6075820abef182880e3772e9f6fbd087caef3f82166203aff9ea314442e8b0565b8564b9fff82a50163666f6f0263626172036362617a056763617264616e6f066572756c657380"
}
