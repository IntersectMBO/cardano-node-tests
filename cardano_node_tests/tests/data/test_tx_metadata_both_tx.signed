{
    "type": "Tx MaryEra",
    "description": "",
    "cborHex": "83a400818258209d43c61d25e780112200a023840b2fdfdf318c12acc9b230685f788167a9419f00018182581d609545290549fe29e2c3e4905118d625bc0695779814551e948185146e1a002adfda021a0002e6e6075820abef182880e3772e9f6fbd087caef3f82166203aff9ea314442e8b0565b8564ba1008182582008d25b073137a6ea634a9aeb56aa5455955ba490a01eed8a735ac7e6dea51b855840512a0ded73ed8e6d06e578203086cdc967f7f5675abbde73e49864f26ee7e50512d5f4011da96eb69c2936deb1c55e1997bd5f782437a7eec7e99fa1fb49b10a82a50163666f6f0263626172036362617a056763617264616e6f066572756c657380"
}
