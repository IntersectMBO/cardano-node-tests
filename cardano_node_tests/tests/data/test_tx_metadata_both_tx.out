auxiliary data: AuxiliaryDataRaw {txMetadata = fromList [(1,S "foo"),(2,S "bar"),(3,S
  "baz"),(5,S "cardano"),(6,S "rules")], auxiliaryScripts = StrictSeq {fromStrict
  = fromList []}}
auxiliary data hash: AuxiliaryDataHash {unsafeAuxiliaryDataHash = SafeHash "abef182880e3772e9f6fbd087caef3f82166203aff9ea314442e8b0565b8564b"}
certificates: []
era: Mary
fee: 190182
inputs:
- 9d43c61d25e780112200a023840b2fdfdf318c12acc9b230685f788167a9419f#0
mint:
  lovelace: 0
  policies: {}
outputs:
- address:
    Bech32: addr_test1vz2522g9f8lznckrujg9zxxkyk7qd9thnq292855sxz3gmszxqcfc
    credential:
      key hash: 9545290549fe29e2c3e4905118d625bc0695779814551e948185146e
    network: Testnet
    stake reference: StakeRefNull
  amount:
    lovelace: 2809818
    policies: {}
update: null
validity interval:
  invalid before: null
  invalid hereafter: null
withdrawals: []
